loc/LocProject.json

```diff
@@ -0,0 +1,14 @@
+{
+    "Projects": [
+        {
+            "LanguageSet": "VS_Main_Languages",
+            "LocItems": [
+                {
+                    "SourceFile": "./loc/vscode-csharp.xlf",
```

Comment:
`vscode-csharp.xlf` is a generated file from `l10nDevGenerateXlf` task in package.json.
It is passed to the localization build.

./loc folder is used to stage all the generated files.